File: perl/mojoclient.py
import asyncio
import websockets

async def send_message():
    async with websockets.connect('ws://localhost:3000/echo') as websocket:
        await websocket.send('Hello, server!')
        response = await websocket.recv()
        print(f'Received: {response}')

asyncio.get_event_loop().run_until_complete(send_message())

File: perl/MOJO/mojoclient.pl
use Mojo::WebSocket::Client;

my $client = Mojo::WebSocket::Client->new;
$client->connect('ws://localhost/echo' => sub {
    my ($client, $tx) = @_;
    $tx->send('Hello, server!');
    $tx->on(message => sub {
        my ($tx, $msg) = @_;
        print "Received: $msg\n";
    });
});

